AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/preview/related.js

```diff
@@ -101,10 +101,11 @@ define([
             $.ajax({
                 type: 'GET',
                 url: this.preview().relatedImagesUrl,
+                showLoader: true,
                 dataType: 'json',
                 data: {
                     'image_id': record.id,
-                    'limit': this.tabImagesLimit
+                    'limit': this.tabImagesLimit + 1
```

Comment:
Please do not modify the limit on the frontend (in javascript). The frontend should not know the details of related images provider logic
```suggestion
                    'limit': this.tabImagesLimit
```